Omise.Examples/Example.cs

```diff
@@ -28,5 +29,30 @@ public Example()
                 ExpirationYear = DateTime.Now.Year + 10,
             });
         }
+
+        // Creates a new PaymentSource called RetrieveSourceInternetBanking, as sources can be created client-side (as well as server-side).
+        protected async Task<PaymentSource> RetrieveSourceInternetBanking()
```

Comment:
`RetrieveSourceWithType(type OffsiteTypes)` ?

Reply:
Hmmm, I think that's too generic for the example, I wanted to be explicit here to show the usage